lib/pymedphys/_experimental/chartchecks/helpers.py

```diff
@@ -404,6 +404,7 @@ def get_all_treatment_history_data(connection, mrn):
     )
     treatment_history = pd.DataFrame(data=table, columns=columns)
     treatment_history = treatment_history.sort_values(by=["date"])
+    # pylint: disable=unsubscriptable-object,unsupported-assignment-operation
```

Comment:
Unless it is placed on the same line with some code, I believe this results in the ignores happening for the whole file. If you want it for the whole file it's best to put it right at the top of the file.

Reply:
Hmmmm based on my experiments it does seem that these affect only the intended lines. Perhaps due to the indentation? It wasn't possible to put it on the same line due to the line being too long and black moving it down anyway...

Reply:
> Hmmmm based on my experiments it does seem that these affect only the intended lines. Perhaps due to the indentation?

Okay :) I learnt something again :)

> It wasn't possible to put it on the same line due to the line being too long and black moving it down anyway...

This one is possible, when you save it (which should auto-run black), that will then split the brackets, then you just have to move the comment, save again (which should auto-run black), and repeat

But, if it only works for the line or code block in question, then, okay with me as-is.